Update Course Management UI for the Ability To Remove A Course

kozlowskik opened the pull request:
The UI for the web app was updated so that:
- It now contains all of the Add course info in a drop-down menu.
- All delete Course information is now in a drop-down menu.
- Delete a course includes entering the `Course Number` and `Course Title` as well as a `Delete Course` button to send the changes to the backend.
- The contents of issue #31 have been addressed.

Commit:
Update GEEKSLogicalSchema.doc

Update the logical schema. Change the primary key for the Course table from "Name" to the combination of "Name" and "Title"

Commit:
Update the createTables.sql

Primary key in the Course table was changed from "Number" to the combination of "Number" and "Title."

Commit:
Update GEEKSConceptualSchema.pdf

Change the PK of the entity Course from "Name" to the combination of "Name" and "Title"

Commit:
Update GEEKSLogicalSchema.doc

Changed PK in Table and Attribute Summary for Course to "Number, Title"

Commit:
Merge pull request #23 from smacademic/update-course

Update Course Primary Key

kozlowskik committed:
Merge pull request #27 from smacademic/features

Add **Add Course** to the `dev` branch

kozlowskik committed:
Fixed issue with Course Tab Showing Incorrectly

Added the id attribute back into the course management div to hide the content from showing under every showing div. Also corrected the declaration of the tab in the navigation section of the code.   Fixes #31

kozlowskik committed:
Updated Course Mgmt Structure and Added Remove Course

Updated the structure of Add Course so that all of the input fields are now in a drop down menu. Added the Remove Course UI section which is also located in a drop down menu to save screen space if it is not needed. Remove a course contains 2 input fields, one for Number and one for Title along with a Remove Course button to submit the changes.

DasilvaB commented on src/webapp/client/index.html:
```diff
@@ -213,23 +215,56 @@ <h2><center>Course Management</center></h2>
 
-					<a id="btnAddCourse" class="waves-effect waves-light btn">Add Course</a>
-				</div><!--End of add course row.-->
+								<a id="btnAddCourse" class="waves-effect waves-light btn">Delete Course</a>
```
should this be btnRemoveCourse?

kozlowskik replied:
Yes, it should be. Thanks for catching my copy and paste error

kozlowskik committed:
Fixed Remove Course Button ID

Fixed the ID of the Delete course button to have a value of "btnRemoveCourse". It was left as "btnAddCourse" in a copy/paste error.

DasilvaB commented on src/webapp/client/index.html:
```diff
@@ -213,23 +215,56 @@ <h2><center>Course Management</center></h2>
+						<div class="collapsible-body">
+							<div class="row">
+								<div class="input-field col s12 m6 l3">
+									<input id="courseName">
```
So, i dont know a ton about this, but I think if you have more than one element with the same id, for example you have two input boxes named courseName, here and above, when you click the button to send to the server I think it'll send the first element it finds. 

Meaning if someone enters something in the addCourse section then again in the removeCourse section it will send the data thats in the addCourse section when you want to send the data in the removeCourse section.

Least I think thats how it works, I could be wrong though.

DasilvaB reviewed (commented):
Not sure if this is an issue but will bring it up just incase

kozlowskik committed:
Updated Id Attrbutes for add/remove

Updated the ID attributes for add course input fields and remove course input fields to have distanct names from each other to help prevent any kind of errors that could arise due to duplicate ID names for different functions.
